File: v1/src/main/java/com/api/v1/dtos/CustomerResponseDto.java
package com.api.v1.dtos;

import java.time.LocalDate;

public record CustomerResponseDto(
        String fullName,
        String ssn,
        LocalDate birthDate,
        String email,
        String address,
        String phoneNumber,
        String createdAt,
        String updatedAt
) {
}

File: kotlin/v2/src/main/kotlin/com/api/v2/customer/dtos/CustomerResponseDto.kt
package com.api.v2.customer.dtos

import java.time.LocalDate

data class CustomerResponseDto(
    var fullName: String,
    var ssn: String,
    var birthDate: LocalDate,
    var email: String,
    var phoneNumber: String,
    var gender: String,
    var createdAt: String,
    var updatedAt: String?
)
